_resources/shared/emergency-medical/911.md

```diff
@@ -0,0 +1,13 @@
+---
+title: 911
+href: tel:911
+category: emergency-medical
+phones:
+  primary:
+    display: 911
+    link: 911
+---
+
+**While this is not an alternative to calling 911, it may still prove helpful in certain cities if you request "fire and rescue only".**
```

Comment:
Hmm, since we already have the bit at the top about calling 911, I wonder if this is a little redundant (and might possibly undercut community trust in this tool)? IDK I'm not sure that's even founded, just thinking out loud.

Reply:
I had that thought as well. Maybe we should move the "fire and rescue only" suggestion to the top with the callout to call 911 in emergency situations?